### DISCUSSION TITLE
post/2021/2021-03-tmux/

### DISCUSSION THREAD

**Discussion by giscus** [zyrikby/website_related#23] [Announcements]
# post/2021/2021-03-tmux/

Until recently, I used tmux occasionally, only if I had to run some experiments on a remote server and later see the results of the execution. Basically, I used it only as a mean to execute commands in the background. If I needed to run several commands on a remote server parallelly, I used to open several terminals, connect each of them to the remote host and then switch between them.
Recently, I started working with a remote server through ssh more often and the routine, I used to, became very operation consuming. So, to improve my effectiveness, I spend several hours reading articles, watching videos and trainings how to use tmux. This article combines the knowledge I have acquired. It is also a crib for me if I forget something in the future.

https://zhauniarovich.com/post/2021/2021-03-tmux/

**Comment by knowblesse** [comment]
The most clear introduction to the tmux I found on the internet. Thank you for summarizing the key features with figures.

**Reply by zyrikby** [reply]
Thanks @knowblesse! This is the most valuable evaluation of the article I would like to receive! Part of my job is to teach, therefore, I would like to be very good at explaining complex things!

**Comment by carpincho988** [comment]
Thank you for the very concise and laconic explanation of tmux, I have ever read!
Спасибо большое вам, за ваш труд!
